src/ostorlab/runtimes/cloud/runtime.py

```diff
@@ -86,11 +87,14 @@ def scan(
             None
         """
         try:
+            # Support multiple link assets for local runtime for the cloud runtime.
             if len(assets) > 1:
-                raise NotImplementedError()
-            else:
+                if isinstance(assets[0], link.Link):
+                    asset = assets
+            elif len(assets) == 1:
                 asset = assets[0]
-            # we support multiple assets for local runtime for the cloud runtime. we take just the first asset.
+            else:
+                raise NotImplementedError()
```

Comment:
if all(isintance(a, link.Link) for a in assets) is True:
  asset = assets
elif len(assets) > 1:
  loggert.warning('Assets has more, the one, scan will only target first asset'
  asset = assets[0]
else:
  asset = assets[0]